pkgs/development/julia-modules/default.nix

```
@@ -167,6 +167,7 @@ runCommand "julia-${julia.version}-env" {
 
   inherit julia;
   inherit juliaWrapped;
+  version = julia.version;
   meta = julia.meta;
```

Would be nice to also inherit pname a switch over to use inherit.


```suggestion
   inherit (julia) version pname meta;
```